android/engine/src/test/java/org/smartregister/fhircore/engine/rulesengine/RulesFactoryTest.kt

```diff
@@ -506,4 +509,33 @@ class RulesFactoryTest : RobolectricTest() {
       }
     return carePlan
   }
+
+  @Test
+  fun testPrettifyDateReturnXDaysAgo() {
+    val daysAgo = 2
+    val inputDateString = LocalDate.now().minusDays(daysAgo - 1).toString()
```

Comment:
I'm kinda confused by why the minus 1 is necessary, I guess if you think of from the end of today? But that feels not intuitive to a user, or maybe there's some more flakiness here that I'm not seeing